src/test/java/uk/gov/hmcts/reform/civil/service/PaymentDateServiceTest.java

```diff
@@ -0,0 +1,82 @@
+package uk.gov.hmcts.reform.civil.service;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.hmcts.reform.civil.model.CaseData;
+import uk.gov.hmcts.reform.civil.model.RespondToClaim;
+import uk.gov.hmcts.reform.civil.model.RespondToClaimAdmitPartLRspec;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(SpringExtension.class)
+public class PaymentDateServiceTest {
+
+    @Mock
+    private DeadlinesCalculator calculator;
+
+    @InjectMocks
+    private PaymentDateService paymentDateService;
+
+    @Test
+    void shouldGetWhenWillThisAmountBePaid_whenWhenWillThisAmountBePaidArePresent() {
+        //Given
+        LocalDate whenWillPay = LocalDate.now().plusDays(5);
+        CaseData caseData = CaseData.builder()
+            .respondToClaimAdmitPartLRspec(
+                RespondToClaimAdmitPartLRspec.builder()
+                    .whenWillThisAmountBePaid(whenWillPay)
+                    .build()
+            )
+            .build();
+        //When
+        LocalDate payDate = paymentDateService.getPaymentDateAdmittedClaim(caseData);
+        //Then
+        org.assertj.core.api.Assertions.assertThat(payDate).isEqualTo(whenWillPay);
```

Comment:
Would be more readable to static import assert methods

Reply:
Done. Thank you for the comments